tests/errors/e_cstruct.pyx

```diff
@@ -24,7 +24,7 @@ cdef void eggs(Spam s):
 
 
 _ERRORS = u"""
-7:39: C struct/union member cannot be a Python object
+7:4: C struct/union member cannot be a Python object
```

Comment:
Interesting. This should say 5 instead of 4, but 39 was definitely wrong, by all means.

Reply:
I think this is OK given 0 indexing. The arrow comes out like:

```
    obj             # error - py object
    ^
------------------------------------------------------------
```

(although I did change the arrow position in this PR since I don't think it was 0-based)